plugins/test/fu-plugin-test.c

```diff
@@ -160,19 +160,37 @@ fu_plugin_update (FuPlugin *plugin,
 		return FALSE;
 	}
 	fu_device_set_status (device, FWUPD_STATUS_DECOMPRESSING);
-	for (guint i = 1; i <= 100; i++) {
-		g_usleep (1000);
-		fu_device_set_progress (device, i);
+	guint64 decompress_ms = 100;
+	const gchar *decompress_ms_str = g_getenv("FWUPD_PLUGIN_TEST_DECOMPRESS_MS");
+	if (decompress_ms_str != NULL) {
+		decompress_ms = g_ascii_strtoll(decompress_ms_str, NULL, 10);
+	}
```

Comment:
@djcampello is it possible to specify "no braces for single line if's" in the clang-format somehow?

Reply:
My understanding is that this goes beyond what `clang-format` can do but it may be possible to achieve with `clang-tidy`